src/cryptsetup/cryptsetup-tokens/cryptsetup-token-util.h

```diff
@@ -11,14 +12,20 @@
 #define crypt_log_error(cd, ...) crypt_logf(cd, CRYPT_LOG_ERROR,  __VA_ARGS__)
 #define crypt_log(cd, ...)       crypt_logf(cd, CRYPT_LOG_NORMAL, __VA_ARGS__)
 
-#define crypt_log_debug_errno(cd, e, ...) ({ \
+#define crypt_log_lvl_errno_internal(cd, e, lvl, ...) ({ \
```

Comment:
doesn't matter, but we usually call functions/macros that take the "full" number of arguments we support (in contrast to wrappers that fill in some for you) with a _full suffix. hence maybe name crypt_log_full_errno()?

but really doesn't matter